### PR TITLE
Add unity layout & ferryman page

### PR DESCRIPTION
**Pull request by shamshirz**
I just jammed that shizz in there. I think it's a decent first attempt at putting a unity game on a completely unstyled page.

### CODE REVIEW

File: _drafts/yyyy-mm-dd-game-template.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (rjdellecese) [inline _drafts/yyyy-mm-dd-game-template.html:1]: Did you mean to commit this file? What it fo'?

**Review by rjdellecese — APPROVED** [review]
Wowowow look ah dah, :shipit:

File: _drafts/yyyy-mm-dd-game-template.html (hunk 1 above)
Comment (shamshirz) [reply _drafts/yyyy-mm-dd-game-template.html:1]: I did, but now I'm like…is it useful? probably not, but I don't feel strongly 1 way or the other

**Comment by rjdellecese** [comment]
🙌

**Pull request merged**